plugins/glustershd/actor.go

```diff
@@ -0,0 +1,88 @@
+package glustershd
+
+import (
+	"errors"
+
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/glusterd2/xlator"
+        "github.com/gluster/glusterd2/plugins/afr"
+	gderrors "github.com/gluster/glusterd2/pkg/errors"
+)
+
+var names = [...]string{"replicate", "afr"}
+
+type shdActor struct{}
+
+func (actor *shdActor) Do(v *volume.Volinfo, key string, value string) error {
+	glustershDaemon, err := newGlustershd()
+	if err != nil {
+		return err
+	}
```

Comment:
This actor will be called for all `replicate.*` or `afr.*` options. Add condition to return nil if key is not selfHealKey